.github/workflows/build-preview.yml

```diff
@@ -112,7 +112,7 @@ jobs:
       if: steps.upload.outcome == 'success'
       run: node ./bin/github-actions/postLinksToDetails.js
       env:
```

Comment:
This script now adds links that are 404:
https://download.delta.chat/desktop/preview/deltachat-desktop-preview-release-2.6.0.AppImage

Reply:
That's due to the changes of dashes to dots.

Reply:
It takes either `FULL_ARTIFACT_URL` or `PR_BRANCH`. Works now.